### ISSUE TITLE
type numeric answer - chrome

### ISSUE THREAD

**Issue by magvt** [expfactory/expfactory-experiments#425]
The screen isn't accepting any response - from laptop or extern
![numeric response](https://cloud.githubusercontent.com/assets/17258830/14463923/cc52c822-009a-11e6-8ad9-12742a35db13.jpg)
al keyboard.

**Comment by vsoch** [comment]
@IanEisenberg added some kind of keyboard tracking to the experiments, and I think this would lead to the error you are seeing. Let's wait for his report on what he did.

**Comment by vsoch** [comment]
Does the pop up appear continuously no matter what you do, and when you click "ok" ?

**Comment by magvt** [comment]
The error msg appeared only when I clicked NEXT.

**Comment by vsoch** [comment]
Ok, so the order of operations is:

- see page
- click Next (without answering)
- pop up appears
- pop up won't go away / keyboard /mouse not responsive?

**Comment by magvt** [comment]
The order is:

·        See page

·        Try to type in response

·        Click Next

·        Pop up appears

I didn’t try to close the pop up.

**Comment by vsoch** [comment]
Ok, so you were unable to type anything in the box before hitting next, or clicking next cleared your answer from the box?

**Comment by vsoch** [comment]
Could you please look in the console for errors? (right click on screen --> view source --> console tab). I just tested this experiment in the same battery, and after I get the error message I'm able to type in the box and continue:

![image](https://cloud.githubusercontent.com/assets/814322/14465737/e265d3fa-0088-11e6-916e-139771196ce4.png)

**Comment by vsoch** [comment]
It could be something specific to windows that was not tested.

**Comment by magvt** [comment]
I was unable to type anything in the box, and so I hit next in order to proceed.  That’s when the pop up appeared,.

**Comment by magvt** [comment]
I closed out that experiment – and had to restart my laptop so I think it’s lost.
If you can get me back to it, I’d be happy to.

**Comment by vsoch** [comment]
Sure thing, [here you go](https://expfactory.org/batteries/5/702f9c8b-db57-4c54-b5ce-6d3804fa4fa2/accept)

**Comment by magvt** [comment]
OK, here you go:
1) screen 1 - I cannot enter a number in.  I've attached the screen shot and console text.
2) I hit next, got the error msg.  I could not get to the console when the pop up was showing.  I attached screen shot.
3) I closed the pop up and copied the console text. 

[enter number - screen 1 console.txt](https://github.com/expfactory/expfactory-experiments/files/215361/enter.number.-.screen.1.console.txt)
![enter number - screen 1](https://cloud.githubusercontent.com/assets/17258830/14466421/c2c1c3da-00a4-11e6-98bb-d8f67fbe3ff6.jpg)
[enter number - screen  after hitting next - console.txt](https://github.com/expfactory/expfactory-experiments/files/215363/enter.number.-.screen.after.hitting.next.-.console.txt)

**Comment by vsoch** [comment]
Thanks! Sorry I gave the wrong instruction - you want to do:

- right click
- inspect element
- then select the "console" tab - it looks like this:

![image](https://cloud.githubusercontent.com/assets/814322/14466922/ac89aedc-008d-11e6-9a93-37c34fa30667.png)

Some browsers will show it on the right side of the page as well. We are looking for text in red that would indicate an error.

**Comment by IanEisenberg** [comment]
There is no keyboard tracking in this experiment. Nothing has been changed and other subjects have successfully completed it.

From the console output, she is using an old version of the cognitive reflection task, even though it has been updated on expfactory.org. (And was updated days ago). Not sure why.

**Comment by vsoch** [comment]
@IanEisenberg please see https://github.com/expfactory/expfactory-docker/issues/110 I have confirmed the commit in expfactory is identical to the current:

https://github.com/expfactory/expfactory-surveys/commit/e9541effadd4cb9fee557848aaf3e409b5f8f628

and that the survey.tsv files are identical between the one installed in expfactory, and the one pulled from a github this morning. Could you please clarify what you see to reflect this being an old version?

**Comment by IanEisenberg** [comment]
You're correct, looking back at the html the experiment is up to date. Hm, not sure what's going on given that the task works for me and other users on Windows and Mac using chrome.

**Comment by vsoch** [comment]
@IanEisenberg @magvt is this still an issue?
